src/Lucene.Net.ObjectMapping.Tests/DictionaryTest.cs

```diff
@@ -191,10 +191,12 @@ public void QueryOnDictionary()
             const int NumObjects = 10;
 
             Write(NumObjects);
-            Assert.AreEqual(NumObjects, writer.MaxDoc());
+            Assert.AreEqual(NumObjects, writer.MaxDoc);
 
-            using (Searcher searcher = new IndexSearcher(dir, true))
+            using (var reader = DirectoryReader.Open(dir))
```

Comment:
Please don't use 'var', but always use the specific type for better readability/maintenance of code